database/buildSchema/35_file.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE public.file (
     original_filename varchar NOT NULL,
     user_id integer REFERENCES public.user (id) ON DELETE CASCADE,
     template_id integer REFERENCES public.template (id) ON DELETE CASCADE,
-    application_serial varchar REFERENCES public.application (serial) ON DELETE CASCADE,
+    application_serial varchar REFERENCES public.application (serial) ON DELETE CASCADE ON UPDATE CASCADE,
```

Comment:
We should have done this a long time ago. Without it, if you change the application serial and there's a file that points to it (via serial), it'll fail with an error. Now it'll update along with it :)